indico/types/model_group.py

```diff
@@ -24,3 +27,44 @@ class ModelGroup(BaseType):
     status: str
     selected_model: Model
     task_type: str
+
+
+class ModelTaskType(Enum):
+    """A list of valid task types for a model group."""
+    CLASSIFICATION = 1
+    FORM_EXTRACTION = 2
+    OBJECT_DETECTION = 3
+
+class ModelType(Enum):
+    STANDARD = 1
+    FINETUNE = 2
+    OBJECT_DETECTION = 3
+    FORM_EXTRACTION = 4
+    DOCUMENT = 5
```

Comment:
We also use `TFIDF_LR` and `TFIDF_GBT` model types for classification. The default classification type is `TFIDF_GBT`

```suggestion
    DOCUMENT = 5
    TFIDF_LR = 6
    TFIDF_GBT = 7
```